_posts/hi/newsletters/2022-08-31-newsletter.md

```diff
@@ -0,0 +1,140 @@
+
+- [<!--when-did-bitcoin-core-deprecate-the-mining-function?--> Bitcoin Core ने माइनिंग फंक्शन को कब बंद कर दिया?]({{bse}}114687)
+  Peter Wullie ने पिछले कुछ वर्षों में Bitcoin Core के भीतर खनन से संबंधित
+  सुविधाओं का एक ऐतिहासिक अवलोकन प्रदान किया है।
+
+- [<!--utxo-spendable-by-me-or-deposit-to-exchange-after-5-years?--> UTXO मेरे द्वारा खर्च किया जा सकता है या 5 साल बाद Exchange के लिए जमा किया जा सकता है?]({{bse}}114901)
+  Stickies-v Bitcoin स्क्रिप्ट ऑपरेटरों का एक सिंहावलोकन प्रदान करता है, कैसे
+  [taproot][topic taproot] ​​[MAST][topic mast] के साथ सक्षम है
+  गोपनीयता और शुल्क के नजरिए से खर्च करने की स्थिति में सुधार करता है, और बताता है
+  कि स्क्रिप्ट की [वाचाओं][topic covenants] की कमी प्रस्तावित शर्तों को पूरी तरह से स्क्रिप्ट
+  में असंभव बना देती है। Vojtěch Strnad बताते हैं कि पूर्व-हस्ताक्षरित लेनदेन प्रस्तावित खर्च की
+  शर्तों को पूरा करने में मदद कर सकते हैं।
+
+- [<!--what-was-the-bug-for-the-bitcoin-value-overflow-in-2010?--> 2010 में Bitcoin मूल्य अतिप्रवाह के लिए बग क्या था?]({{bse}}114694)
+  Andrew Chow [मूल्य अतिप्रवाह बग][value overflow bug] और इसके कई मुद्रास्फीति प्रभावों को सारांशित
+  करता है: बड़े आउटपुट के साथ-साथ गलत गणना लेनदेन शुल्क।
+
+## रिलीज और रिलीज उम्मीदवार
+
+*लोकप्रिय Bitcoin इन्फ्रास्ट्रक्चर परियोजनाओं के लिए नए रिलीज और रिलीज उम्मीदवार। कृपया नई रिलीज़ में
+अपग्रेड करने या रिलीज़ उम्मीदवारों का परीक्षण करने में मदद करने पर विचार करें।*
+
+- [LND 0.15.1-beta][] एक रिलीज़ है जिसमें "[शून्य कॉन्फ़ चैनल][topic zero-conf channels],
+  SCID [उपनाम][aliases], और हर जगह [taproot][topic taproot] पते इस्तेमाल
+  करने के लिए समर्थन शामिल है"।
+
+## उल्लेखनीय कोड और दस्तावेज़ीकरण परिवर्तन
+
+*इस सप्ताह [Bitcoin Core][bitcoin core repo], [Core Lightning][core lightning repo],
+[Eclair][eclair repo], [LDK][ldk repo], [LND][LND Repo] में उल्लेखनीय परिवर्तन।
+[libsecp256k1][libsecp256k1 repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][Rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[BDK][bdk repo], [Bitcoin Improvement Proposals (BIP)][BIPs repo], और
+[Lightning BOLTs][bolts repo]।*
+
+- [Bitcoin Core #23202][] एक [PSBT][topic psbt] बनाने की क्षमता के साथ `psbtbumpfee` RPC
+  का विस्तार करता है, जो शुल्क लेन-देन को प्रभावित करता है, भले ही लेन-देन के कुछ या सभी इनपुट वॉलेट संबंधित
+  न हों। PSBT को फिर उस वॉलेट के साथ साझा किया जा सकता है जो उस पर हस्ताक्षर कर सकता है।
+
+- [Eclair #2275][] एक [दोहरी वित्त पोषित][topic dual funding] LN सेटअप लेनदेन शुल्क बंपिंग के लिए
+  समर्थन जोड़ता है। PR नोट करता है कि, इस PR के साथ, "दोहरी फंडिंग पूरी तरह से Eclair द्वारा समर्थित है!"
+  हालांकि यह भी नोट करता है कि डिफ़ॉल्ट रूप से दोहरी फंडिंग अक्षम है और [Core Lightning के साथ क्रॉस संगतता][news143 cln df]
+  के लिए परीक्षण भविष्य में जोड़े जाएंगे।
+
+- [Eclair #2387][] [signet][topic signet] के लिए समर्थन जोड़ता है।
+
+- [LDK #1652][][onion message] के लिए अद्यतन समर्थन [topic onion messsage] * उत्तर पथ *
```

Comment:
@shaavan the [onion message] ...... [topic onion messsage] needs to be corrected so it is clear what the link is and what is the link text. Im unsure of the language so need your help with that.

Also note "onion message" vs "onion mess**s**age"

Reply:
Hi @bitschmidty, I was on a mini break, so I got your message today. Apologies for the late reply.

I figured the link tag was not correctly placed; hence the problem was not detected while running the `make production -j8`. There was also a similar mistake following it. So I have corrected both of them.
Thanks, @bitschmidty, for catching them!